integration/README.md

```diff
@@ -17,10 +17,20 @@ npx playwright install
-| `npm run codegen`                 | `playwright codegen localhost:1111`                                       | Runs Playwright codegen for integration-tests. Must have local server running on port 1111 before starting. |
-| `screenshots:light`               | `playwright test --config=visual-diff.config.ts`                          | Take screenshots of images on light theme. Primarily for use on Github Actions.                             |
-| `screenshots:all`                 | `cross-env FULL_DIFF=true playwright test --config=visual-diff.config.ts` | Take screenshots of images on light, dark, and high-contrast themes. Primarily for use on Github Actions.   |
+| Command                     | Referent                                                                  | Description                                                                                                 |
+| --------------------------- | ------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `npm run integration`       | `playwright test --config=integration-tests.config.ts`                    | Runs all tests in the `/integration-tests` folder.                                                          |
+| `npm run integration:debug` | `playwright test --debug --config=integration-tests.config.ts`            | The same as above but in Playwright debug mode.                                                             |
+| `npm run codegen`           | `playwright codegen localhost:1111`                                       | Runs Playwright codegen for integration-tests. Must have local server running on port 1111 before starting. |
+| `screenshots:light`         | `playwright test --config=visual-diff.config.ts`                          | Take screenshots of images on light theme. Primarily for use on Github Actions.                             |
+| `screenshots:all`           | `cross-env FULL_DIFF=true playwright test --config=visual-diff.config.ts` | Take screenshots of images on light, dark, and high-contrast themes. Primarily for use on Github Actions.   |
+
+## Accessibility testing with Axe and Playwright
+
+Although we must rely primarily on careful functional testing and code review to ensure a high standard of accessibilty, we've also adopted automated accessibilty with [Playwright and AXE](https://playwright.dev/docs/accessibility-testing) testing in pipelines to ensure we don't forget to write accessible components and documentation examples.
+
+Some major things to remember are:
+
+- We keep our list of pages on which we run accessibility checks in [accessibility.spec.ts](https://github.com/microsoft/atlas-design/tree/main/integration/tests/accessibility.spec.ts). If you've added a new documentation page, please to ensure it is added to the list
```

Comment:
please to ensure ->  please ensure that?

Reply:
updated!